tests/integration/test_registration_interface.py

```diff
@@ -0,0 +1,71 @@
+"""
+These tests implement the OMA-ETS-LightweightM2M-V1_1-20190912-D specification:
+https://www.openmobilealliance.org/release/LightweightM2M/ETS/OMA-ETS-LightweightM2M-V1_1-20190912-D.pdf
```

Comment:
This link should be moved to a more "generic" place.

Reply:
moved to README.md